tests/run_code_style.sh

```diff
@@ -4,13 +4,11 @@ set -xeu
 
 if [ $1 = "lint" ]; then
     flake8 ignite tests examples --config setup.cfg
-    isort . --check --settings setup.cfg
-    black . --check --config pyproject.toml
+    ufmt diff .
 elif [ $1 = "fmt" ]; then
-    isort . --settings setup.cfg
-    black . --config pyproject.toml
+    ufmt format .
 elif [ $1 = "mypy" ]; then
     mypy --config-file mypy.ini
 elif [ $1 = "install" ]; then
-    pip install flake8 "black==21.12b0" "isort==5.7.0" "mypy"
+    pip install flake8 "black==21.12b0" "usort==1.0.1" "ufmt==1.3.1.post1" "mypy==0.910"
```

Comment:
Why do you fix mypy to 0.910 ?

Reply:
It was pinned in `run_code_style.bat` as well, so I just assumed it was intended to have pinned versions for everything. Happy to change it back to unpinned if that's desired.

Reply:
oh, I see, we forgot to unpin it there. Yes, please unpin it.

Reply:
Done!